### PR TITLE
[bitnami/mongodb] enable backup with auth: {enabled: false}

### PR DESCRIPTION
**Pull request by sfhl**
### Description of the change

This will enable backups with `auth: {enabled: false}`

### Checklist

- [X] Chart version bumped in `Chart.yaml` according to [semver](http://semver.org/). This is *not necessary* when the changes only affect README.md files.
- [X] Variables are documented in the values.yaml and added to the `README.md` using [readme-generator-for-helm](https://github.com/bitnami-labs/readme-generator-for-helm)
- [X] Title of the pull request follows this pattern [bitnami/<name_of_the_chart>] Descriptive title
- [X] All commits signed off and in agreement of [Developer Certificate of Origin (DCO)](https://github.com/bitnami/charts/blob/main/CONTRIBUTING.md#sign-your-work)

### CODE REVIEW

**Comment by javsalgar** [comment]
Hi!

Thank you so much for the contribution! Could you please fix the DCO issue?

**Comment by sfhl** [comment]
I am so sorry, DCO should be fixed now.

File: bitnami/mongodb/templates/backup/cronjob.yaml [hunk 1]
```diff
@@ -83,10 +85,14 @@ spec:
             {{- if .Values.backup.cronjob.command }}
             command: {{- include "common.tplvalues.render" (dict "value" .Values.backup.cronjob.command "context" $) | nindent 14 }}
             {{- else }}
-            command: 
+            command:
               - /bin/sh
               - -c
+            {{- if .Values.auth.enabled }}
               - "mongodump --username=${MONGODB_ROOT_USER} --password=${MONGODB_ROOT_PASSWORD} --host=${MONGODB_SERVICE_NAME} --port=${MONGODB_PORT_NUMBER} ${MONGODB_CLIENT_EXTRA_FLAGS} --oplog --gzip --archive=${MONGODUMP_DIR}/mongodump-$(date '+%Y-%m-%d-%H-%M').gz"
```
Comment (joancafom) [inline bitnami/mongodb/templates/backup/cronjob.yaml:92]: I would rather prefer the changes to be performed in a one-liner here, as the rest of the command is identical and this way we would prevent duplicity:

```suggestion
              - "mongodump {{- if .Values.auth.enabled }} --username=${MONGODB_ROOT_USER} --password=${MONGODB_ROOT_PASSWORD} {{- end }} --host=${MONGODB_SERVICE_NAME} --port=${MONGODB_PORT_NUMBER} ${MONGODB_CLIENT_EXTRA_FLAGS} --oplog --gzip --archive=${MONGODUMP_DIR}/mongodump-$(date '+%Y-%m-%d-%H-%M').gz"
```

**Review by joancafom — COMMENTED** [review]
Thanks for your PR! Could you please take a look at my comment?

File: bitnami/mongodb/templates/backup/cronjob.yaml (hunk 1 above)
Comment (sfhl) [reply bitnami/mongodb/templates/backup/cronjob.yaml:92]: that's  fine for me

**Pull request merged**